auditbeat/docs/fields.asciidoc

```diff
@@ -4849,6 +4849,16 @@ type: long
 Process parent id.
 
 
+--
+
+*`process.pgid`*::
```

Comment:
As proposed in your ECS PR, I would go for `process.group.id` here (and `process.user.id` if you have that)

Reply:
Actually seeing the full context, I wonder if we shouldn't simply map to the top level fields, like you did for user.name. So:

- `system.process.username` => `user.name`
- `system.process.pgid` => `group.id`

And equivalent mappings, if you have user id and group name.

WDYT @ruflin ?

I'd say unless the events can contain more than one user or group, we shouldn't nest them.

Reply:
As commented in https://github.com/elastic/ecs/pull/311, the `pgid` identifies a group of processes, not a group of users. It is a value similar to `ppid`, I think it belongs to process.